walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -3,16 +3,18 @@
     <head>
       <meta charset="utf-8">
       <link rel="stylesheet" href="style.css">
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+      <script type="text/javascript" src="script.js"></script>
       <title>Raymond's Portfolio</title>
     </head>
     <body>
       <div class="header-container">
         <a class="header-item" href="#home">Raymond C. Alvarez Velez</a>
-        <a class="header-item" href="#about">About</a>
-        <a class="header-item" href="#work">My Work</a>
-        <a class="header-item" href="#contact">Contact</a>
-      </div>
-      <div class="content-container">
+        <a class="header-item" href="#about" onclick="animateScrollTop('about-container')">About</a>
+        <a class="header-item" href="#work" onclick="animateScrollTop('work-container')">My Work</a>
+        <a class="header-item" href="#contact" onclick="animateScrollTop('footer-container')">Contact</a>
```

Comment:
onclick attributes are generally considered bad practice because of mixing of template and logic, more difficult to test (there's probably a bunch more, just Google it)...but if you won't be building on this code, I'm ok with it
I'll leave it up to you